File: common_config/src/main/java/cn/woochen/common_config/mvp/FragmentPresenterProxyImpl.java
package cn.woochen.common_config.mvp;


/**
 *fragment代理类
 *@author woochen123
 *@time 2018/1/19 11:55
 *@desc
 */
public class FragmentPresenterProxyImpl<V extends IBaseView> extends PresenterProxyImpl<V> {
    public FragmentPresenterProxyImpl(V view) {
        super(view);
    }
}

File: common_config/src/main/java/cn/woochen/common_config/mvp/FragmentPresenterProxyImpl.kt
package cn.woochen.common_config.mvp


import cn.woochen.common_config.mvp.proxy.PresenterProxyImpl

/**
 * fragment代理类
 * @author woochen123
 * @time 2018/1/19 11:55
 * @desc
 */
class FragmentPresenterProxyImpl<V : IBaseView>(view: V) : PresenterProxyImpl<V>(view)
